turborepo-tests/integration/turbo.json

```diff
@@ -2,7 +2,7 @@
   "extends": ["//"],
   "pipeline": {
     "test": {
-      "dependsOn": ["cli#build"],
+      "dependsOn": ["cli#build", "^build"],
```

Comment:
This should be handled by adding a synthetic task:

```
  "pipeline": {
    "topo": {
      "dependsOn": ["^topo"]
    },
    "test": {
      "dependsOn": ["cli#build", "^topo"],
      "env": ["GO_TAG"]
    }
```

This pattern also enables scoping down the `inputs` for any part of the synthetic graph.

Reply:
done